src/logger.js

```diff
@@ -0,0 +1,8 @@
+﻿//
+// Logger
+//
+
+const winston = require( "winston" );
+
+winston.handleExceptions( new winston.transports.Console( { colorize: true } ) );
```

Comment:
in my opinion, it would be nice to see in the logs and the execution time ... 
`  timestamp: true`